.github/workflows/dev-environment-pipeline.yml

```diff
@@ -0,0 +1,119 @@
+name: Dev Environment Pipeline
+
+on:
+  workflow_call:
+    outputs:
+      docker_image_tag:
+        description: "The Docker image tag that was deployed"
+        value: ${{ jobs.build.outputs.DOCKER_IMAGE_TAG }}
+    secrets:
+      ARTIFACTORY_NUBIA_USERNAME:
+        required: true
+      ARTIFACTORY_NUBIA_TOKEN_DEVELOPER:
+        required: true
+      DEV_SEPOLIA_URL:
+        required: true
+      TEST_ACCOUNT_ADDRESS:
+        required: true
+      TEST_ACCOUNT_PRIVATE_KEY:
+        required: true
+
+env:
+  DOCKER_REGISTRY: nethermind.jfrog.io
+  REPO_DEV: nubia-oci-local-dev
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    outputs:
+      DOCKER_IMAGE_TAG: ${{ steps.set_tag.outputs.DOCKER_IMAGE_TAG }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Define image tag
+        id: set_tag
+        run: |
+          export DOCKER_IMAGE_TAG=$(git describe --tags)
+          echo "DOCKER_IMAGE_TAG=$DOCKER_IMAGE_TAG" >> $GITHUB_ENV
+          echo "DOCKER_IMAGE_TAG=$DOCKER_IMAGE_TAG" >> $GITHUB_OUTPUT
+
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Build Docker Image
+        uses: docker/build-push-action@v6
+        with:
+          context: .
+          platforms: "linux/amd64"
+          push: false
+          outputs: type=docker,dest=/tmp/juno-image.tar
```

Comment:
maybe the problem could be that the type is set to docker instead of OCI, let me branch off this and play around